### PR TITLE
Qualcomm AI Engine Direct - Fix aihub path failing due to memory planning pass

### PR DESCRIPTION
**Pull request by winskuo-quic**
## Summary
Update MemoryPlanningPass to use default greedy algo.

### CODE REVIEW

**Comment by winskuo-quic** [comment]
Hi @cccclai,
Newest Mainline is causing QAIHUB flow to fail due to change of interface for MemoryPlanningPass.
Please have a look.
Thanks

**Comment by winskuo-quic** [comment]
Hi @cccclai,

I wanted to follow up on this PR, which all QAIHub scripts will fail in current mainline.
Your feedback is greatly appreciated.

Thanks for your time!

**Comment by cccclai** [comment]
Hi sorry was in the conference for the past few days. Any chance we know why it's failing? When does it start to fail?

File: examples/qualcomm/qaihub_scripts/utils/utils.py [hunk 1]
```diff
@@ -68,7 +68,6 @@ def gen_pte_from_ctx_bin(
     for pte_name in pte_names:
         print(f"{pte_name} generating...")
         memory_planning_pass = MemoryPlanningPass(
-            memory_planning_algo="greedy",
```
Comment (cccclai) [inline examples/qualcomm/qaihub_scripts/utils/utils.py:71]: [The default algo is greedy](https://github.com/pytorch/executorch/blob/e12b37e83885342b7664f7434a3988ab5c6cc111/exir/passes/memory_planning_pass.py#L30) and I'm not sure how this line will fix the issue. Could you elaborate a bit more?

**Comment by cccclai** [comment]
Actually nvm, just saw https://github.com/pytorch/executorch/pull/4727

**Review by cccclai — APPROVED** [review]

**Comment by winskuo-quic** [comment]
> Hi sorry was in the conference for the past few days. Any chance we know why it's failing? When does it start to fail?

All good and thanks a lot for reviewing.
 As you mentioned above, https://github.com/pytorch/executorch/pull/4727 is causing the failure.

**Comment by cccclai** [comment]
hey mind rebasing the PR? Somehow the tests are not running

**Comment by winskuo-quic** [comment]
> hey mind rebasing the PR? Somehow the tests are not running

Just rebased.
Thanks

**Pull request closed without merging**